src/commands/new.ts

```diff
@@ -12,40 +12,123 @@ import {
   stopSpinner,
   clearSpinners,
 } from "../tools/pretty"
+import { normalize } from "../tools/normalize"
 
 // CLI tool versions we support
 const deps: { [k: string]: string } = {
   podInstall: "0.1",
 }
 
+export interface Options {
+  /**
+   * Log raw parameters for debugging, run formatting script not quietly
+   *
+   * Input Source: `prompt.confirm` | `parameter.option`
+   * @default false
+   */
+  debug?: boolean
+  /**
+   * Remove existing directory otherwise throw if exists
+   *
+   * Input Source: `prompt.confirm` | `parameter.option`
+   * @default false
+   */
+  overwrite?: boolean
+  /**
+   * Input Source: `parameter.option`
+   * @deprecated flag left in for backwards compatability, warn them to use old Ignite
+   * @default undefined
+   */
+  boilerplate?: string
+  /**
+   * alias for `boilerplate`
+   *
+   * Input Source: `parameter.option`
+   * @deprecated flag left in for backwards compatability, warn them to use old Ignite
+   * @default undefined
+   */
+  b?: string
+  /**
+   * custom bundle identifier (android only)
+   *
+   * Input Source: `prompt.ask` | `parameter.option`
+   * @default `com.${name}`
+   * @example 'com.pizzaapp'
+   */
+  bundle?: string
+  /**
+   * Input Source: `parameter.option`
+   * @deprecated this option is deprecated. Ignite sets you up to run native or Expo
+   * @default undefined
+   */
+  expo?: boolean
+  /**
+   * Package manager to install dependencies with
+   *
+   * Input Source: `prompt.ask`| `parameter.option`
+   *
+   * Default: packager that executed current command, i.e. `npx ignite new`, `yarn dlx ignite new`, `pnpm dlx ignite new`
+   */
+  packager?: "npm" | "yarn" | "pnpm"
+  /**
+   * Create new git repository and create an inital commit with boilerplate changes
+   *
+   * Input Source: `prompt.confirm` | `parameter.option`
+   * @default true
+   */
+  git?: boolean
+  /**
+   * Whether or not to run packager install script after project is created
+   *
+   * @default true
+   */
+  installDeps?: boolean
+  /**
+   * React Native Colo Loco is no longer installed with Ignite,
+   * but we will give instructions on how to install it if they pass in `--colo-loco`   *
+   *
+   * Input Source: `prompt.confirm` | `parameter.option`
+   * @default false
+   */
+  coloLoco?: boolean
+}
+
 export default {
   run: async (toolbox: GluegunToolbox) => {
+    // #region Toolbox
```

Comment:
Let's remove the #region marks.

Reply:
See: https://github.com/infinitered/ignite/pull/1999#discussion_r948481135